src/commands/download_folder.ts

```diff
@@ -26,10 +27,10 @@ new CLICommand({
 			});
 			const driveId = await ardrive.getDriveIdForFolderId(folderId);
 			const driveKey = await parameters.getDriveKey({ driveId: driveId });
-			await ardrive.downloadPrivateFolder({ folderId, destFolderPath, maxDepth, driveKey });
+			await ardrive.downloadPrivateFolder({ folderId, destFolderPath, customFolderName, maxDepth, driveKey });
 		} else {
 			const ardrive = cliArDriveAnonymousFactory({});
-			await ardrive.downloadPublicFolder({ folderId, destFolderPath, maxDepth });
+			await ardrive.downloadPublicFolder({ folderId, destFolderPath, customFolderName, maxDepth });
 		}
 		console.log(`Folder with ID "${folderId}" was successfully download to "${destFolderPath}"`);
```

Comment:
We are not logging the fill path here. 
```shell
$ yarn ardrive download-folder -f "47f5bde9-61ba-49c7-b409-1aa4a9e250f6" --max-depth 1 -l ./my-awesome-folder
Folder with ID "47f5bde9-61ba-49c7-b409-1aa4a9e250f6" was successfully download to "/home/matibat/ardriveapp/ardrive-cli"
```
Suggestion:
```suggestion
		const fullOutputPath = customFolderName ? `${destFolderPath}/${customFolderName}` : destFolderPath;
		console.log(`Folder with ID "${folderId}" was successfully download to "${fullOutputPath}"`);
```